routes/api/resources.js

```diff
@@ -52,4 +52,38 @@ router.post('/habits/:habit_id',
     }
 );
 
+router.put('/:id',
+    passport.authenticate('jwt', { session: false }),
+    (req, res) => {
+        Resource.findById(req.params.id).then((resource) => {
```

Comment:
https://mongoosejs.com/docs/api.html#model_Model.findByIdAndUpdate

you don't have to use this, for now, but this is here for future reference